qcodes/instrument/group_parameter.py

```diff
@@ -55,23 +55,18 @@ def __init__(self,
         self._initial_value = initial_value
```

Comment:
`self.group` should be a read only property

Reply:
agree!